package.json

```diff
@@ -1,98 +1,57 @@
 {
-  "name": "forklift-console-plugin",
-  "version": "0.0.1",
+  "name": "@monorepo/root",
   "private": true,
+  "version": "0.0.1",
   "repository": "git@github.com:kubev2v/forklift-console-plugin.git",
   "license": "Apache-2.0",
+  "workspaces": [
+    "./packages/*"
+  ],
   "scripts": {
-    "clean": "rm -rf dist",
-    "build": "yarn clean && NODE_ENV=production yarn webpack",
-    "start": "yarn webpack serve",
+    "clean": "rm -rf dist && rm -rf coverage && npm run clean -ws",
+    "clean:all": "npm run clean && rm -rf node_modules && npm run clean:all -ws",
+    "start": "npm run start -w @kubev2v/forklift-console-plugin",
```

Comment:
`start` fails if run after `clean:all`. It seems that it requires `build` first.

Reply:
> start fails if run after clean:all. It seems that it requires build first.

yes, the plugin now depends on the libraries of the monorepo, to start the plugin a user needs first to build the monorepo libraries.

I added a `npm run build:dependencies` to build them, but maybe we need something stronger ?
I can add `build:dependencies` before each `start` but then it may build libraries that don't need to be re-build.

action:
I will add a `start:plugin` script that will only start the plugin.
I will change `start` to also build the dependencies

for future improvement:
make the webpack dev server listen to changes in the libs and build them on the fly

Reply:
added the `start:plugin` script, `start` now also build the dependencies